src/infohudmodes.asm

```diff
@@ -735,8 +736,16 @@ status_vspeed:
   .newjump
     LDA #$0001 : STA !ram_roomstrat_counter : STA !ram_walljump_counter
 
+    ; Print initial jump speed over item%
+    LDA $0B38 : BNE +
```

Comment:
Is this check necessary?  Why not always print the speed if we are here? (jump just pressed)

Reply:
Wall jumps and space jumps have always predictable starting speeds. Normal jump height can be a bit unpredictable with speed booster equipped and dash held (with or without boots). I don't want the value to disappear if you have to wall jump or space jump after the jump that needs to be measured.